Dockerfile

```diff
@@ -14,4 +14,5 @@ RUN pip install -r /app/requirements.txt
 # With `docker compose`, dev. directory already mounted at /app
 #COPY . .
 
+#CMD ["/bin/cat", "/dev/zero"]
```

Comment:
can remove this command if not in use

Reply:
I thought I had removed that several commits ago.  I guess I was mistaken.